### PR TITLE
Added a command line calculator function

### PR DESCRIPTION
**Pull request by namantw**
Fixes https://github.com/rahulkumaran/Utlyz-CLI/issues/10

I tested the calc function on my ubuntu 16.04, everything is working.

### CODE REVIEW

File: Utilities/calc.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+def cli(add,sub,mul, div, mod):
+    	if(add): #for addition
+    		print a+b
+    	elif(sub): #for subtraction
+    		print a-b
+    	elif(mul): #for multiplication
+    		print a*b
+    	elif(div): #for division
+    		print a/b
+    	elif(mod): #for moulus
```
Comment (rahulkumaran) [inline Utilities/calc.py]: can you change the spelling of modulus here

File: Utilities/setup.py [hunk 2]
```diff
@@ -30,6 +31,7 @@
 		'news=news:cli',
 		'football=football:cli',
 		'xkcd=xkcd:cli',
+		'calc=calc:cli'
 		]
 	},
 )
```
Comment (rahulkumaran) [inline Utilities/setup.py]: Good job here!
Glad that you made the necessary changes to setup.py too

File: Utilities/calc.py (hunk 1 above)
Comment (rahulkumaran) [reply Utilities/calc.py]: Everything looks perfect apart from that!

**Comment by namantw** [comment]
@rahulkumaran done 👍

**Pull request merged**